packages/react-vapor/src/components/collapsible/CollapsibleSelectors.ts

```diff
@@ -0,0 +1,13 @@
+import * as _ from 'underscore';
+
+import {IReactVaporState} from '../../ReactVapor';
+import {CollapsibleState} from './CollapsibleReducers';
+
+const isExpanded = (state: IReactVaporState, id: string): boolean => {
+    const collapsible: CollapsibleState = _.findWhere(state.collapsibles, {id});
+    return collapsible?.expanded;
```

Comment:
`return !!collapsible?.expanded;`
to make sure you always return a boolean if you get an undefined